style.css

```diff
@@ -1 +1,357 @@
+@import /*  Add your styling here */
+  url('https://fonts.googleapis.com/css2?family=Cookie&family=Montserrat&display=swap');
 /*  Add your styling here */
+:root {
+  --green: #15f1a7;
+  --yellow: #edf50c;
+  --green-light: #8be5f3;
+  --red-light: #ffcab0;
+  --blue-light: #83cee0;
+  --purple: #ea6af6;
+  --pink: #ef61cb;
+  --black: #000;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
```

Comment:
I don't think you need grid layout at this level. Remember, if you use block elements like `div` and `section`, they will naturally sit one below another and form a column, you don't need to add any CSS to achieve that.